src/widgets.ts

```diff
@@ -93,6 +128,20 @@ export abstract class Widget<
     this.teardownIframe()
   }
 
+  /**
+   * Uses matching to get our url targetOrigin, or falls back to our widgets url
+   */
+  private getTargetOrigin() {
```

Comment:
Pretty sure this could just be a private getter: `private get targetOrigin(): string`